src/tests/Directory.Build.targets

```diff
@@ -113,6 +113,24 @@
     <SkipImportILTargets Condition="'$(CLRTestBuildAllTargets)' != '' And '$(CLRTestNeedTarget)' != '$(CLRTestBuildAllTargets)'">true</SkipImportILTargets>
   </PropertyGroup>
 
+  <Target Name="CopyMergedWrapperReferences"
+      Condition="'$(IsMergedTestRunnerAssembly)' == 'true'"
+      AfterTargets="Build"
+      BeforeTargets="CopyNativeProjectBinaries">
+    <PropertyGroup>
+          <CopyMergedWrapperReferencesHasRun>true</CopyMergedWrapperReferencesHasRun>
+    </PropertyGroup>
```

Comment:
It doesn't look like this property is used. Do we need to set it?
```suggestion
```

Reply:
Thanks Jeremy for spotting, I have removed the superfluous property I originally used as local instrumentation when debugging the change in 2nd commit on this PR.

Reply:
If you prefer triggering this action off another build target or rolling in into an existing one, I have no problems with it whatsoever, I have actually only started testing this change on a larger scale and I have mostly randomly put the code next to the existing logic for publishing native product items.

Reply:
(Just for context, this accounts for the two remaining failures apart from the six ones caused by the ilproj rename logic.)

Reply:
I think leaving it as-is for this change just to get us off the ground is reasonable.

We have a lot of complexity around how we support copying the native binaries around (due to the fact that we can do the copy as a separate step from the build), so I'm fine holding off doing a more invasive change and just getting in something simple to unblock.